modules/accredited_representative_portal/app/services/filter_service.rb

```diff
@@ -0,0 +1,48 @@
+class FilterService
```

Comment:
let's replace this with a more descriptive name like POARequestSearchFilterService